routeDetails.html

```diff
@@ -0,0 +1,101 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<style>
+		table {background-color: linen; font-family: "Verdana", Times, serif;}
+		th {background-color: #4CAF50;color: white;}
+		tr:nth-child(even) {background-color: #f2f2f2}
+		p {font-family: "Verdana", Times, serif;}
+	</style>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+	<title>ArcTouch Code Challenge</title>
+	<script type="text/javascript" src="prototype.js"></script>
+	<script type="text/javascript" src="utils.js"></script>
+	<script type="text/javascript">
+		//<![CDATA[
+
+		var routeId   = getURLParameterByName("routeId");
+		var routeNo   = getURLParameterByName("routeNo");
+		var routeLine = getURLParameterByName("routeLine");
+		var street    = getURLParameterByName("street");
+
+		function getRouteDetails() {
+			var oOptions1 = {

```

Comment:
I got the example from internet and forgot to change it to a better name.

2016-03-02 17:26 GMT-03:00 André Luiz Cardoso <notifications@github.com>:

> In routeDetails.html
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1#discussion_r54785256>
> :
>
> > +		p {font-family: "Verdana", Times, serif;}
> > +	</style>
> > +	<meta name="viewport" content="width=device-width, initial-scale=1" />
> > +	<title>ArcTouch Code Challenge</title>
> > +	<script type="text/javascript" src="prototype.js"></script>
> > +	<script type="text/javascript" src="utils.js"></script>
> > +	<script type="text/javascript">
> > +		//<![CDATA[
> > +
> > +		var routeId   = getURLParameterByName("routeId");
> > +		var routeNo   = getURLParameterByName("routeNo");
> > +		var routeLine = getURLParameterByName("routeLine");
> > +		var street    = getURLParameterByName("street");
> > +
> > +		function getRouteDetails() {
> > +			var oOptions1 = {
>
> Are you sure oOptions1 and oOptions2 are good names?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1/files#r54785256>
> .
>